FakeNFT/Scenes /UserDetail/UserDetailPresenter.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import UIKit
+
+protocol UserDetailPresenterProtocol: AnyObject {
+    func loadUserDetails()
+    func openWebsite()
+    func openCollection()
+}
+
+final class UserDetailPresenter: UserDetailPresenterProtocol {
+    
+    weak var view: UserDetailViewProtocol?
+    private let interactor: UserDetailInteractorProtocol
+    private let router: UserDetailRouterProtocol
+    private let userId: String
+    private var userDetail: UserDetail?
+    
+    init(view: UserDetailViewProtocol, interactor: UserDetailInteractorProtocol, router: UserDetailRouterProtocol, userId: String) {
```

Comment:
Вот тут то же самое, линт также будет ругаться, поскольку строка не должна привышать 128 символов в длинну 
```suggestion
    init(
        view: UserDetailViewProtocol, 
        interactor: UserDetailInteractorProtocol, 
        router: UserDetailRouterProtocol, 
        userId: String
    ) {
```